lib/services/lwm2mHandlers/commonLwm2m.js

```diff
@@ -88,24 +86,28 @@ function observeActiveAttributes(payload, registeredDevice, callback) {
     }
 
     for (var i = 0; i < activeAttributes.length; i++) {
-        var lwm2mMapping = {};
+        // Ezequiel: esto debe ser nulo, y todos los if han sido modificados
```

Comment:
Please, briefly explain why the `if` modification. Is there any case where the previous form didn't work and the present do?